src/textual/message.py

```diff
@@ -90,9 +91,17 @@ def _set_forwarded(self) -> None:
         """Mark this event as being forwarded."""
         self._forwarded = True
 
-    def _set_sender(self, sender: MessagePump) -> None:
-        """Set the sender."""
+    def set_sender(self, sender: MessagePump) -> Self:
```

Comment:
Could use a little more detail in the docstring. I doubt the app developer would know what the sender was.